packages/fast-components-react-msft/src/listbox-item/listbox-item.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Listbox item",
+    "description": "A listbox item component's schema definition.",
+    "type": "object",
+    "id": "listbox-item",
+    "properties": {
+        "disabled": {
+            "title": "Disabled",
+            "type": "boolean"
+        },
+        "displayString": {
+            "title": "Display string",
+            "type": "string"
+        },
+        "id": {
+            "title": "Unique ID",
+            "type": "string"
+        },
+        "value": {
+            "title": "Value",
+            "type": "string"
+        }
```

Comment:
arent id and value required?

Reply:
yep

Reply:
added required items in the base select pr before pulling this one.